presto-docs/src/main/sphinx/connector/inmemory.rst

```diff
@@ -0,0 +1,55 @@
+==================
+InMemory Connector
+==================
+
+The InMemory connector stores all data and metadata in RAM on workers
+and both are discarded when Presto restarts.
+
+.. warning::
+
+    This connector is in early experimental stage it is not recommended
+    to use it on a production environment.
+
+.. warning::
+
+    This connector does not support ``DROP TABLE`` or any other method
+    to release held resources. Once table is populated it will consume
+    memory until next Presto restart.
```

Comment:
What if only Coordinator is restarted but Workers are not? Is metadata lost but data is still occupying Workers' memory is essentially orphaned?
What happens if Coordinator is not restarted but all Workers are? Table will be "empty" or in invalid state?